src/index.tsx

```diff
@@ -66,7 +66,7 @@ export const useClearCache = (props?: OwnProps) => {
 
     // clear browser cache and reload page
     await setVersion(version || latestVersion);
-    window.location.reload(true);
+    window.location.replace(window.location.pathname + window.location.search + window.location.hash);
```

Comment:
`location.reload` is not deprecated. Please check on this doc. https://www.quanzhanketang.com/jsref/met_loc_reload.html#:~:text=The%20reload()%20method%20is,reload(true).

Reply:
Please check the current spec: https://www.w3.org/TR/html50/browsers.html#dom-location-reload
The argument is not longer in it.